### ISSUE TITLE
Couldn't import movie

### ISSUE THREAD

**Issue by ginjohn5snaps** [Radarr/Radarr#1544]
I just startes getting this error.
The log tells me that it could not import because the destination already exists. When I check the destination folder, in fact the folder does exist, and the movie is also there and renamed correctly.
In the downloads folder, the file is also there still, not renamed.

Why is this?

**Comment by ginjohn5snaps** [comment]
My Radarr is running under Windows 10. It worked perfectly up until yesterday. Sonarr is also still working great.

The destination for my files are a NAS drive, set up with UNC paths in Radarr. 

The only thing I did in between it not working is change a bit in my network environment. (NAS and Windows PC were both connected directly to my internet  modem/router, now my Windows PC is connected to a subsequent router in the network. This and I changed the PC name. None of these things should matter?)
